app/commands/git/pull_request_review/create.rb

```diff
@@ -0,0 +1,16 @@
+module Git
+  class PullRequestReview
+    class Create
+      include Mandate
+
+      initialize_with :pull_request, :node_id, :attributes
+
+      def call
+        ::Git::PullRequestReview.create_or_find_by!(node_id: node_id) do |r|
```

Comment:
Might it be better to call this `pr_id`?

Reply:
Let's discuss the naming in a chat later.